<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;宋体&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chongkai">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>380</y>
      <width>150</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;幼圆&quot;;</string>
    </property>
    <property name="text">
     <string>重开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>180</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>与电脑对战</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_226">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>240</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>玩家对战</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_227">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>450</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>结束当前游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_228">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>300</y>
      <width>150</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>玩扫雷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saolms1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>730</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>9x9/10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saolms2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>730</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>12x12/20</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saolms3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>730</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>12x12/35</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>380</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>-10</y>
      <width>16</width>
      <height>771</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_port">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_listen">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>监听</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_server">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>281</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>640</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_server">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>620</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIP">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnect">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="link">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>当前连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fuwuqi">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>570</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>服务器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kehuduan">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>570</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>客户端</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_229">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>650</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>网络对战准备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_230">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>510</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>显示网络模块</string>
    </property>
   </widget>
   <widget class="QLabel" name="background2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>0</y>
      <width>301</width>
      <height>761</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/new/prefix1/22233.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
